docs/source/Links.md

```diff
@@ -2,127 +2,121 @@
-27](https://www.defcon.org/html/defcon-27/dc-27-index.html) hacker conference in Paris. This is an
-Evennia game dir with batchcode to build the custom _Hackers_ style cyberspace zone with puzzles and
-challenges [used during the conference](https://dcdark.net/home#).
-- [Arx sources](https://github.com/Arx-Game/arxcode) - Open-source code release of the very popular
-[Arx](https://play.arxmush.org/) Evennia game. [Here are instructions for installing](Arxcode-
-installing-help)
-- [Evennia-wiki](https://github.com/vincent-lg/evennia-wiki) - An Evennia-specific Wiki for your
-website.
-- [Evcolor](https://github.com/taladan/Pegasus/blob/origin/world/utilities/evcolor) - Optional
-coloration for Evennia unit-test output.
-- [Paxboards](https://github.com/aurorachain/paxboards) - Evennia bulletin board system (both for
-telnet/web).
-- [Encarnia sources](https://github.com/whitehorse-io/encarnia) - An open-sourced game dir for
-Evennia with things like races, combat etc. [Summary
-here](https://www.reddit.com/r/MUD/comments/6z6s3j/encarnia_an_evennia_python_mud_code_base_with/).
-- [The world of Cool battles sources](https://github.com/FlutterSprite/coolbattles) - Open source
-turn-based battle system for Evennia. It also has a [live demo](http://wcb.battlestudio.com/).
-- [nextRPI](https://github.com/cluebyte/nextrpi) - A github project for making a toolbox for people
-to make [RPI](https://www.topmudsites.com/forums/showthread.php?t=4804)-style Evennia games.
-- [Muddery](https://github.com/muddery/muddery) - A mud framework under development, based on an
-older fork of Evennia. It has some specific design goals for building and extending the game based
-on input files.
-- [vim-evennia](https://github.com/amfl/vim-evennia) - A mode for editing batch-build files (`.ev`)
-files in the [vim](https://www.vim.org/) text editor (Emacs users can use [evennia-
+
+## Evennia Community
+
+- [Evennia Game Index](http://games.evennia.com/) - An automated listing of Evennia games.
+- [Evennia official Discord channel](https://discord.gg/AJJpcRUhtF)
+- [Evennia official IRC
+channel](https://webchat.freenode.net/?channels=evennia&uio=MT1mYWxzZSY5PXRydWUmMTE9MTk1JjEyPXRydWUbb) ( #evennia at [irc.freenode.net:6667](http://irc.freenode.net:6667/) )
+- [Evennia official forums](https://github.com/evennia/evennia/discussions) on Github Discussions.
+- [Evennia subreddit](https://www.reddit.com/r/Evennia/)
+
+## Third-party Evennia tools
+
+- [Discord relay](https://github.com/InspectorCaracal/evennia-things/tree/main/discord_relay) - Two-way chat relays between Evennia channels and Discord channels.
+- [Evcolor](https://github.com/taladan/Pegasus/blob/origin/world/utilities/evcolor) - Optional coloration for Evennia unit-test output.
+- [Evennia-docker](https://github.com/gtaylor/evennia-docker) - Evennia in a [Docker container](https://www.docker.com/) for quick install and deployment in just a few commands.
```

Comment:
It would be better to refer to this as `Evennia with docker-compose` since it does not build a new docker image but uses Evennia's official image. It would be worth to also link to Evennia's own Docker documentation: https://www.evennia.com/docs/latest/Running-Evennia-in-Docker.html